simpeg_drivers/electricals/direct_current/two_dimensions/__init__.py

```diff
@@ -14,6 +14,3 @@
 #  not be provided or otherwise made available to any other person.
 #
 # ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
-
-
-from .params import DirectCurrent2DParams
```

Comment:
restore

Reply:
Why?

Reply:
Ok so changing them all is bigger undertaking. To be dealt with on GEOPY-1495